system/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM solita/ubuntu-systemd
+
+ARG uid=1000
+
+# Install environment
+RUN apt-get update -y && apt-get install -y \
+	git \
+	wget \
+	python3.5 \
+	python3-pip \
+	python-setuptools \
+	python3-nacl \
+	apt-transport-https \
+	ca-certificates \
+	ssh \
+	vim \
+	mc
```

Comment:
it makes sense to add `&& rm -rf /var/lib/apt/lists/*` here as well to remove apt cache reducing that docker layer

Reply:
Done.